source/stylesheets/_navigation.scss

```diff
@@ -0,0 +1,210 @@
+
+// ==== main nav =====
+.nav-title{
+	color: $text-color-lighter;
+	font-weight: $medium-weight;
+	border-bottom: none;
+	padding: $medium-padding;
+}
+
+ul{
```

Comment:
Be careful, this rule will apply to all `ul` elements (not just in the navigation). This applies to the `li` and `li a` rules below as well. You may want to make a class instead, like `.list--unstyled` (which removes the bullets, spacing, etc).